src/components/auth/AuthLayout.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types'
+import { Menu } from "lucide-react"
+import { Link } from 'react-router-dom'
+import { useState } from 'react'
+import { CalendarCheck } from 'lucide-react'
+const AuthLayout = ({ children }) => {
+    const [state, setState] = useState(false)
+    const menus = [
```

Comment:
make a separate file called header constant

Reply:
Moved the data to another file